internal/controller/http/redfish/v1/middleware.go

```diff
@@ -0,0 +1,54 @@
+// Package v1 provides Redfish v1 API middleware.
+package v1
+
+import (
+	"strings"
+
+	"github.com/coreos/go-oidc/v3/oidc"
+	"github.com/gin-gonic/gin"
+	"github.com/golang-jwt/jwt/v5"
+
+	"github.com/device-management-toolkit/console/config"
+)
+
+// RedfishJWTAuthMiddleware returns a Gin middleware that validates JWT tokens
+// and returns Redfish-compliant error responses for authentication failures.
+func RedfishJWTAuthMiddleware(jwtKey string, verifier *oidc.IDTokenVerifier) gin.HandlerFunc {
```

Comment:
This would not be needed for the initial verion as would be supporting basic auth.
JWTAuthMiddleware could be consider when we implement Sessions and Account Service